libcodechecker/analyze/analyzers/result_handler_plist_to_db.py

```diff
@@ -53,19 +53,35 @@ def __store_bugs(self, files, reports, client):
             hasher = sha256()
             hasher.update(source)
             content_hash = hasher.hexdigest()
-            file_descriptor = client.needFileContent(file_name, content_hash)
+            file_descriptor = client.needFileContent(file_name,
+                                                     content_hash,
+                                                     self.__run_id)
             file_ids[file_name] = file_descriptor.fileId
 
             if file_descriptor.needed:
                 source64 = base64.b64encode(source)
                 res = client.addFileContent(content_hash,
                                             source64,
-                                            Encoding.BASE64)
+                                            Encoding.BASE64,
+                                            self.__run_id)
```

Comment:
Why do we need the run id for adding the file content and querying whether we need the file content? The file content should be independent of the runs (because they are deduplicated across runs).